pgcli/pgexecute.py

```diff
@@ -289,7 +289,9 @@ def execute_normal_sql(self, split_sql):
         cur.execute(split_sql)
 
         try:
```

Comment:
Now that there is a while loop, let's get rid of the try/except clause. The while loop condition takes care of safe access.